CHANGELOG.md

```diff
@@ -2,14 +2,303 @@
 
 ## HEAD
 
-* Add TLS support for MySQL: https://github.com/google/trillian/pull/3593
+## v1.6.1
+
+* Recommended go version for development: 1.22
+  * This is the version used by the cloudbuild presubmits. Using a
+    different version can lead to presubmits failing due to unexpected
+    diffs.
+
+### MySQL
+
+* Add TLS support for MySQL by @fghanmi in https://github.com/google/trillian/pull/3593
   * `--mysql_tls_ca`: users can provide a CA certificate, that is used to establish a secure communication with MySQL server. 
   * `--mysql_server_name`: users can provide the name of the MySQL server to be used as the Server Name in the TLS configuration.
+* dedup leafidentityhash values ahead of SQL lookup of existing leaves by @bobcallaway in https://github.com/google/trillian/pull/3607
 
-## Notable Changes
+### Documentation
+
+* Add instructions for using docker to regen derived files by @mhutchinson in https://github.com/google/trillian/pull/3489
+
+### Misc
+
+* Fix invalid Go toolchain version by @roger2hk in https://github.com/google/trillian/pull/3491
+* Replace deprecated `prune-whitelist` flag with `prune-allowlist` for `kubectl` command by @roger2hk in https://github.com/google/trillian/pull/3307
+* Remove @pphaneuf from CODEOWNERS by @roger2hk in https://github.com/google/trillian/pull/3516
+* Don't bump to MySQL 9 until we explicitly choose to by @mhutchinson in https://github.com/google/trillian/pull/3560
+* Don't update to MySQL 9.0 by @mhutchinson in https://github.com/google/trillian/pull/3584
 
-* Updated go version 1.20 -> 1.22
-* Bump kaniko (used to build Docker images in GCB) to 1.20.0
+### Dependency updates
```

Comment:
IMO most of this section is uninteresting. I would like to see the bumps to docker containers and github actions at least omitted from here, leaving only the go deps.

Reply:
Agreed that they're not that interesting. I don't feel too strongly to remove them as they're indeed part of the change in this release, and it takes some more effort to filter them.